charts/images.yaml

```diff
@@ -26,7 +26,7 @@ images:
 - name: csi-provisioner
   sourceRepository: https://github.com/kubernetes-csi/external-provisioner
   repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: v1.6.0
+  tag: v3.0.0
```

Comment:
I think this version should be applied to k8s version >= v1.20.0. Please set the targetVersion field. https://kubernetes-csi.github.io/docs/external-provisioner.html.

And have you verified that whether AliCloud CSI plugin work well with external provisioner v3.0.0?

Reply:
Hi @jia-jerry,
Alicloud plugin is now supporting external provisioner v3.0.0 after fixing this issue: https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver/issues/523

I also added targetVersion field.

Please let me know if there is anything else that should be done.